ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlServerMain.java

```diff
@@ -46,6 +50,10 @@ public static void main(final String[] args) {
       );
 
       final String installDir = properties.getOrDefault("ksql.server.install.dir", "");
+      final String streamsStateDirPath = properties.getOrDefault("ksql.streams.state.dir",
```

Comment:
This is where I was saying we should use `KsqlConfig.KSQL_STREAMS_PREFIX + StreamsConfig.STATE_DIR_CONFIG` instead of hard-coding `ksql.streams.state.dir` :)

Reply:
Aha, I changed it in the error message only. Fixed it :)